news/2021-08-17-pp-sr-survey.md

```diff
@@ -12,9 +12,11 @@ In case you're not clued in, development of osu!'s scoring systems happens mostl
 
 In recent times, there have been conflicting opinions about what the path forward should look like. To help clear the air, we're putting out a short survey to better discover the expectations that all of you have for how things should be.
 
+Please also note that this survey mainly involves the **osu!** game mode for now. We will also be issuing a private version of this survey to a smaller audience of higher skilled players to ensure that we're covering the entire community.
```

Comment:
```suggestion
Please also note that this survey mainly involves the **osu!** game mode for now.

We will also be issuing a private version of this survey to a smaller audience of higher skilled players to give insight into the opinions of both top-end players and also the wider "casual" community.
```

Something like this? Also not sure if we need an additional sentence telling top players that they should respond in this survey regardless.. is that implied enough?